portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,124 @@
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  width:1000px;
```

Comment:
nit: Let's add spaces between the ":" and the CSS property values in this file (e.g. `width: 1000px` instead of `width:1000px`).